csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -529,145 +536,156 @@ private void validateRunResultData(Tensor<float> resultTensor, float[] expectedO
         private void ThrowWrongInputName()
         {
             var tuple = OpenSessionSqueezeNet();
-            var session = tuple.Item1;
-            var inputData = tuple.Item2;
-            var tensor = tuple.Item3;
-            var inputMeta = session.InputMetadata;
-            var container = new List<NamedOnnxValue>();
-            container.Add(NamedOnnxValue.CreateFromTensor<float>("wrong_name", tensor));
-            var ex = Assert.Throws<OnnxRuntimeException>(() => session.Run(container));
-            Assert.Contains("Invalid Feed Input", ex.Message);
-            session.Dispose();
+            using (var session = tuple.Item1)
```

Comment:
Better to follow this pattern as session will not be disposed if an exception is encountered before Dispose() is hit